src/components/FindMovie/FindMovie.tsx

```diff
@@ -16,42 +76,74 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-dander"
+              className={classNames(
+                'input',
+                {
+                  'is-danger': isSearched && !movie,
+                },
+              )}
+              value={query}
+              onChange={(event) => {
+                setQuery(event.target.value);
+                setIsSearched(false);
+              }}
```

Comment:
Create separate methods instead of creating inline event handlers all over the code.
It is important to lead this rule in all tasks and in future test tasks.